test/types/types.test.ts

```diff
@@ -1,8 +1,9 @@
 import { Tester } from '@mighty-justice/tester';
 import { EMPTY_FIELD } from '@mighty-justice/utils';
 
-import { Card, fillInFieldConfig, FormCard, TYPES } from '../../src';
+import { Card, fillInFieldConfig, FormCard, IFieldConfig, TYPES } from '../../src';
 import { TYPE_GENERATORS, valueRenderPairs } from '../factories';
+import { IValue } from '../../src/props';
```

Comment:
is this similar to importing IInjected for possible future use case? IValue not used anywhere else

Reply:
NOPE! This is a mistake you found :)